lab_4/ngrams/ngram_trie.py

```diff
@@ -18,14 +18,8 @@ def _fill_n_grams(self):
         if not isinstance(self.encoded_text, tuple):
             raise ValueError
 
-        n_grams = []
-        for i in range(len(self.encoded_text) - self.size + 1):
-            n_grams.append(tuple(self.encoded_text[i:i + self.size]))
-        self.n_grams = tuple(n_grams)
-
-        for word in self.encoded_text:
-            if (word, ) not in self.uni_grams:
-                self.uni_grams[(word, )] = self.encoded_text.count(word)
+        self.n_grams = self.get_tuple_n_grams()
```

Comment:
I do not understand this change. Is it so necessary? I thought ngrams module is untouchable for students)

Reply:
The lint check is unsatisfied with the number of public methods in this class as well as in others (0/2).